packages/@lwc/engine-core/src/framework/template.ts

```diff
@@ -120,6 +142,8 @@ function buildParseFragmentFn(
     return (strings: string[], ...keys: number[]) => {
         const cache = create(null);
 
+        registerFragmentCache(cache);
```

Comment:
Is there a perf difference between always calling a maybe-noop vs conditionally calling only when defined?
```suggestion
        if (process.env.NODE_ENV === 'test-karma-lwc') registerFragmentCache(cache);
```
```suggestion
        // Assuming we don't initialize as a noop
        registerFragmentCache?.(cache);
```

Reply:
There's no difference; it will get tree-shaken out either way.

It's really six-of-one-half-dozen-of-the-other, but I'm just following the convention we've followed elsewhere for `test-karma-lwc`.

Reply:
```ts
const x = noop;
x();
```
That gets tree-shook? Fancy!